packages/babel-compat-data/README.md

```diff
@@ -2,7 +2,7 @@
 
 > 
 
-See our website [@babel/compat-data](https://babeljs.io/docs/babel-compat-data) for more information.
+See our website [@babel/compat-data](https://babeljs.io/docs/v8-migration-api#babelcompat-data) for more information.
```

Comment:
Thank you. However, this link is not supposed to be the docs of babel-compat-data. To address this issue we will have to actually create a doc page for `babel-compat-data`.

Reply:
@JLHwung Oh I understood. Should I go ahead and close this issue, then?